modules/signals/src/index.ts

```diff
@@ -1 +1,3 @@
-export default 0;
+export { selectSignal } from './select-signal';
+export { signalState } from './signal-state';
+export { SignalStateUpdater } from './signal-state-models';
```

Comment:
Does this type need to be exported?

Reply:
Yes, it can be used to avoid manual typing when creating custom updaters. For example:

```ts
import { SignalStateUpdater } from '@ngrx/signals';

export function addOne<T>(entity: T): SignalStateUpdater<{ entities: T[] }> {
  return (state) => ({ entities: [...state.entities, entity] });
}

// usage:
import { signalState } from '@ngrx/signals';

type State = { entities: User[] };

const usersState = signalState<State>({ entities: [] });

usersState.$update(
  addOne({ name: 'Marko' }),
  addOne({ name: 'Brandon' })
);
```